core/domain/suggestion_services_test.py

```diff
@@ -1345,6 +1351,83 @@ def test_get_reviewable_translation_suggestions(self):
         expected_language_code_list = ['hi', 'hi', 'pt']
         self.assertEqual(actual_language_code_list, expected_language_code_list)
 
+    def test_get_reviewable_translation_suggestions_with_valid_exp_ids( # pylint: disable=line-too-long
+            self):
+        # Add few translation suggestions in different languages.
+        self._create_translation_suggestion_with_language_code('hi')
+        self._create_translation_suggestion_with_language_code('hi')
+        self._create_translation_suggestion_with_language_code('pt')
+        self._create_translation_suggestion_with_language_code('bn')
+        self._create_translation_suggestion_with_language_code('bn')
+        # Add few question suggestions.
+        self._create_question_suggestion_with_skill_id('skill1')
+        self._create_question_suggestion_with_skill_id('skill2')
+        # Provide the user permission to review suggestions in particular
+        # languages.
+        user_services.allow_user_to_review_translation_in_language(
+            self.reviewer_id_1, 'hi')
+        user_services.allow_user_to_review_translation_in_language(
+            self.reviewer_id_1, 'pt')
+        # Get all reviewable translation suggestions.
+        suggestions = suggestion_services.get_reviewable_suggestions_for_translate_content(
+                self.reviewer_id_1, self.opportunity_summary_ids)
+
+        # Expect that the results correspond to translation suggestions that the
+        # user has rights to review.
+        self.assertEqual(len(suggestions), 3)
+        actual_language_code_list = sorted([
```

Comment:
Prefer not adding logic in tests. Is the result order really not deterministic?

Reply:
I was using the earlier written test as a precedent. And it looks like it sorts to be on the safer side while comparing the list contents in the end of the test. But I have removed the sorting and it works now as well.